package.json

```diff
@@ -22,6 +22,7 @@
     "hardhat": "^2.11.2",
     "hardhat-contract-sizer": "^2.6.1",
     "hardhat-spdx-license-identifier": "^2.0.3",
+    "nock": "^13.3.2",
```

Comment:
I would leave all the dependencies necessary for compilation in `dependencies`, and move all additional ones (like `nock`) to `devDependencies`